Core/HLE/sceFont.cpp

```diff
@@ -645,9 +645,26 @@ static void __LoadInternalFonts() {
 	if (!pspFileSystem.GetFileInfo(fontPath).exists) {
 		pspFileSystem.MkDir(fontPath);
 	}
+	if (pspFileSystem.GetFileInfo("disc0:/PSP_GAME/USRDIR/zh_gb.pgf").exists) {
+		for (size_t i = 0; i < ARRAY_SIZE(fontRegistry); i++) {
```

Comment:
It seems like this adds the first font only, and then bails out... I'm pretty sure that's not the way the PSP works.  Maybe instead there's a problem in the font matching function?

-[Unknown]